tapdb/sqlc/queries/assets.sql

```diff
@@ -709,6 +709,19 @@ FROM asset_proofs
 JOIN asset_info
     ON asset_info.asset_id = asset_proofs.asset_id;
 
+-- name: FetchAssetProofsSizes :many
+WITH asset_info AS (
+    SELECT assets.asset_id, script_keys.tweaked_script_key
+    FROM assets
+    JOIN script_keys
+        ON assets.script_key_id = script_keys.script_key_id
+)
+SELECT asset_info.tweaked_script_key AS script_key, 
+LENGTH(asset_proofs.proof_file) AS proof_file_length
+FROM asset_proofs
+JOIN asset_info
```

Comment:
Instead of joining with a CTE, I think we should just do a normal three-way join here between asset_proofs, assets and script_keys. IMO that's a bit easier to read.